tests/examplefiles/conftest.py

```diff
@@ -18,6 +18,7 @@
 
 import pathlib
 import pytest
+from pathlib import Path
```

Comment:
This change looks accidental.

Reply:
This was removed by pulling from the master branch.